build.xml

```diff
@@ -1010,7 +1013,6 @@ xmlns:cs="antlib:com.puppycrawl.tools.checkstyle.ant">
           <include name="zookeeper-server/**"/>
           <include name="zookeeper-it/**"/>
         </fileset>
-        <fileset file="${java.server.resources.dir}/pom.template"/>
```

Comment:
this is for the bin-package target, you'll also need a new copy task here.

Reply:
Fixed.